src/calculateRentalCost.js

```diff
@@ -22,13 +23,21 @@
  *  - calculateRentalCost(3) === 100
  *  - calculateRentalCost(7) === 230
  *
- * @param {number} days
+ * @param {number} payTotal
  *
  * @return {number}
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const payTotal = days * 40;
+
+  if (days < 3) {
+    return payTotal;
+  } else if (days >= 3 && days < 7) {
+    return payTotal - 20;
+  } else {
+    return payTotal - 50;
+  }
```

Comment:
what a magic numbers 🪄
lets create variables for discount amount and days and then use them
it is better for managing them from one place and for understanding what do they mean

avoid such nesting `else if` conditions
This code is hard to read and maintain

check checklist one more time 😉